scripts/CEE/pcap-collector/metadata.yaml

```diff
@@ -2,17 +2,25 @@ file: script.sh
 name: pcap-collector
 shortDescription: Captures traffic on nodes and produce gzipped pcap file.
 description: |
-  Capture network traffic with tcpdump on a node
+  Capture network traffic with tcpdump on a node and then send the capture to the case SFTP server and ticket.
   
-  The response is a gzipped pcap file.
+  This script will only work if a secret exists in the managed scripts namespace that contains a valid single-use
+  SFTP token.
 
-  Examples:
-    # Capture traffic on node ip-10-0-253-170.ap-southeast-2.compute.internal for 10 mins
-    ocm backplane managedjob create CEE/pcap-collector -p TIME=600 -p NODE=ip-10-0-253-170.ap-southeast-2.compute.internal
-    
-    # Getting the resulting pcap file
-    ocm backplane managedjob logs <JOBNAME> | gunzip > node.pcap
-author: John Roche
+  How to use:
+    0. Get the case number from your support case. It is the case_id from this URL format. https://access.redhat.com/support/cases/#/case/{case_id}/discussion
+    1. Have your username and password you use to login to access.redhat.com ready.
+    2. Check whether your user is internal or not. This will be true or false. Run `curl -u username https://access.redhat.com/hydra/rest/contacts/sso/username | jq  .isInternal` with the username and password from step 1.
+    2. Run `curl -u username https://access.redhat.com/hydra/rest/v1/sftp/token/upload/temporary` with your username and password from Step 1.
+    3. Add the token returned in Step 2 to this secret (and delete it if it already exists in the namespace):
+      oc create secret generic pcap-collector-creds --from-literal=username=username_from_step_1 --from-literal=internal=result_from_step_2 --from-literal=password=token_from_step_3  --from-literal=caseid=support_case_number_step_0 -n openshift-backplane-managed-scripts
```

Comment:
We need make sure CEE & SREP have this perms to create the secret under openshift-managed-scripts namespaces.